lib/mangadex/auth.rb

```diff
@@ -40,7 +40,7 @@ def self.login(username: nil, email: nil, password: nil, &block)
       session = response.dig('token', 'session')
       refresh = response.dig('token', 'refresh')
 
-      mangadex_user = Mangadex::Internal::Request.get('/user/me', headers: { Authorization: session })
+      mangadex_user = Mangadex::Internal::Request.get('/user/me', headers: { Authorization: "Bearer "+session })
```

Comment:
I think it might look better to do string interpolation
```suggestion
      mangadex_user = Mangadex::Internal::Request.get('/user/me', headers: { Authorization: "Bearer #{session}" })
```

Reply:
Nice. Good catch.